Sources/Run/main.swift

```diff
@@ -2,18 +2,23 @@ import App
 import Service
 import Vapor
 
-var config = Config.default()
-var env = try Environment.detect()
-var services = Services.default()
-
-try App.configure(&config, &env, &services)
-
-let app = try Application(
-    config: config,
-    environment: env,
-    services: services
-)
-
-try App.boot(app)
-
-try app.run()
+// The contents of main are wrapped in a do/catch block because any errors that get raised to the top level will crash XCode
+do {
+    var config = Config.default()
+    var env = try Environment.detect()
+    var services = Services.default()
+    
+    try App.configure(&config, &env, &services)
+    
+    let app = try Application(
+        config: config,
+        environment: env,
+        services: services
+    )
+    
+    try App.boot(app)
+    
+    try app.run()
+} catch {
+    print(error)
```

Comment:
this results in successful exit of the application which will cause problems. We need to call `exit(1)` manually or do `fatalError(...)`